fetch/api/resources/utils.js

```diff
@@ -72,3 +72,11 @@ function validateStreamFromString(reader, expectedValue, retrievedArrayBuffer) {
     validateBufferFromString(retrievedArrayBuffer, expectedValue, "Retrieve and verify stream");
   });
 }
+
+// From streams tests
+function delay(milliseconds)
```

Comment:
The rest of this file uses two-space indents and (with one exception) no newline before `{`.